types/nestdb/index.d.ts

```diff
@@ -0,0 +1,245 @@
+// Type definitions for NestDb 2.0
+// Project: https://github.com/louischatriot/nestdb
+// Definitions by: Stefan Steinhart <https://github.com/reppners>
+//                 Anthony Nichols <https://github.com/anthonynichols>
+//                 Alejandro Fernandez Haro <https://github.com/afharo>
```

Comment:
Do these other folks want to be pinged for review on future changes to these forked definitions? 😄